raw_disk_io.cpp

```diff
@@ -1,10 +1,120 @@
 #include "raw_disk_io.hpp"
+#include "thread_pool.hpp"
+#include "buffer_pool.hpp"
 
-namespace ezio {
+using namespace ezio;
+using namespace libtorrent;
```

Comment:
Please don't use this.
it might conflict with same names

Reply:
Our raw_disk_io is still in namespace ezio.
and don't `using namespace libtorrent` but `namespace lt = libtorrent` instead